packages/Select/doc.mdx

```diff
@@ -169,6 +169,31 @@ You can _add_ items by passing the `isCreatable` prop. The returned item will be
   </Form>
 </Playground>
 
+## Creatable with custom create button
+
+Passing a `renderCreateItem` function allows you to format the create button in the list.
+It is a function and receives the input value as argument
+
+<Playground>
+  <Form initialValues={{}}>
+    <ConnectedField
+      component={Select}
+      isCreatable
+      label="Social networks"
+      name="welcome8"
+      options={ITEMS}
+      renderCreateItem={value => {
+        return (
+          <Box alignItems="center" display="flex">
```

Comment:
No love for `<Stack>` 😭

Reply:
For what ? :p

Reply:
Ya plus de code en fait ici 🙈 
```js
<Box alignItems="center" display="flex">
  <AddIcon mr="sm" />
  {`Your internationalized create wording "${value}"`}
</Box>
```

vs

```js
<Stack alignItems="center" direction="row" spacing="sm">
  <AddIcon />
  {`Your internationalized create wording "${value}"`}
</Box>
```

Reply:
Je viens d'essayer avec stack mais malheureusement il aime pas trop les text nodes :(